docs/develop-resources/guides/3-add-mapper.md

```diff
@@ -17,6 +17,8 @@ go run . generate-mapper \
    --api-dir $REPO_ROOT/apis 
 ```
 
+Please use the same proto as your mock GCP tests to avoid mismatch in schema definitions. There are some exceptions where you need to [replace the proto go package](https://github.com/xiaoweim/k8s-config-connector/blob/master/dev/tools/controllerbuilder/pkg/codegen/mappergenerator.go#L132).
```

Comment:
We can recommend users to use the same proto, but maybe not saying "please use ... to avoid mismatch". They can use different protos, as long as the serialized data is the same.

Reply:
I see, thank you!